pep-0680.rst

```diff
@@ -191,82 +210,89 @@ Basing on another TOML implementation
-write API or sponsor a PEP that includes a write API. Since it is hard to change
+PEP: core Python packaging tools, and projects that need to read TOML
+configuration files.
+
+Use cases that involve editing an existing TOML file (as opposed to writing a
+brand new one) are better served by a style preserving library. TOML is
+intended as a human-readable and -editable configuration format, so it's
+important to preserve comments, formatting and other markup. This requires
+a parser whose output includes style-related metadata, making it impractical
+to output plain Python types like ``str`` and ``dict``, and complicating the
+design of the API.
+
+Even without considering style preservation, there are too many degrees of
+freedom in how to design a write API. For example, what default style
+(indentation, vertical and horizontal spacing, quotes, etc) should the library
+use for the output, and how much control should users be given over it?
+How should the library handle input and output validation? Should it support
+serialization of custom types, and if so, how? While there are reasonable
+options for resolving these issues, the nature of the standard library is such
+that we only get "one chance to get it right".
+
+Currently, no CPython core developers have expressed willingness to maintain a
+write API, or sponsor a PEP that includes one. Since it is hard to change
 or remove something in the standard library, it is safer to err on the side of
-exclusion and potentially revisit later.
+exclusion for now, and potentially revisit this again later.
 
-So, writing TOML is left to third-party libraries. If a good API and relevant
-use cases for it are found later, it can be added in a future PEP.
+Therefore, writing TOML is left to third-party libraries. If a good API and
+relevant use cases for it are found later, write support can be added in a
+future PEP.
 
 
 Assorted API details
 --------------------
 
-Types accepted by the first argument of ``tomllib.load``
-^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+Types accepted as the first argument of ``tomllib.load``
+''''''''''''''''''''''''''''''''''''''''''''''''''''''''
 
 The ``toml`` library on PyPI allows passing paths (and lists of path-like
-objects, ignoring missing files and merging the documents into a single object).
-Doing this would be inconsistent with ``json.load``, ``pickle.load``, etc. If we
-agree consistency with other stdlib modules is desirable, allowing paths is
-somewhat out of scope for this PEP. This can easily and explicitly be worked
-around in user code, or a third-party library.
+objects, ignoring missing files and merging the documents into a single object)
+to its ``load`` function. However, allowing this here would be inconsistent
+with the behavior of ``json.load``, ``pickle.load`` and other standard library
+functions. If we agree that consistency with other stdlib modules is desirable,
```

Comment:
Good point; I kept it to avoid repetition with the end of the previous sentence, but I modified it to just elide "with other stdlib modules" and replace it with "here".